doc/source/changed_in_57.rst

```diff
@@ -0,0 +1,263 @@
+.. _changed_in_57:
+
+=============================
+Changed in Percona Server 5.7
```

Comment:
no, they usually go to apt_repo yum_repo docs as they evolve during the product time